src/components/daySelector/_tests_/daySelector.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react-native';
+import DaySelector from '../DaySelector';
+import { Day } from '../../../models/app.model';
+
+describe('DaySelector', () => {
+  const daysOfWeek: Day[] = [Day.Sun, Day.Mon, Day.Tue, Day.Wed, Day.Thu, Day.Fri];
+
+  test('renders days correctly', () => {
+    const { getByText } = render(
+      <DaySelector daysOfWeek={daysOfWeek} selectedDay={null} onSelectDay={() => {}} />
+    );
+
+    daysOfWeek.forEach((day) => {
+      const abbreviatedDay = day.substring(0, 3);
+      expect(getByText(abbreviatedDay)).toBeDefined();
```

Comment:
please tell me what does this test - I would like to know.
why can't u test the actual value that it's formatted properly?